.github/workflows/canary-integration-test.yml

```diff
@@ -77,8 +77,13 @@ jobs:
 
     - name: wait for ceph to be ready
       run: |
-        tests/scripts/validate_cluster.sh
-        kubectl -n rook-ceph get pods
+        tests/scripts/validate_cluster.sh >> pods.txt
+        # kubectl -n rook-ceph get pods 
+        # kubectl -n rook-ceph get pods --field-selector=status.phase==Running  | sed '/^csi/d' | sed -n '1!p' | awk '{print $1}' > pods.txt
```

Comment:
Still testing?

Reply:
just wanted to make sure it works ... doing changes in all jobs.

Reply:
Done.